src/components/Form/CheckAclPermsDocContainerForm.jsx

```diff
@@ -111,7 +110,7 @@ const CheckAclPermsDocContainerForm = () => {
               id="set-acl-to"
               name="setAclTo"
               {...user}
```

Comment:
Since moving over to MUI, the selectedUser seems to only seems to set the placeholder, it doesn't set the value for the TextField. Think we could start replacing the useField hooks and use:

```
const [username, setUsername] = useState("")
```

And where we could set `value={selectedUser.username ?? username}` with `onChange={(e) => setUsername(e.target.value)}`

Should be similar to what we're doing in OidcLoginComponent.

Reply:
Updated. Instead of `selectedUser.username ?? username`, I just default the username state to `selectedUser.username`